### PR TITLE
Util: add ja/cn translation in coverage

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: util/coverage/coverage.js [hunk 1]
```diff
@@ -47,41 +47,59 @@ const exVersionToShortName = {
...
+    cn: '5.X',
   },
 };
 
 const contentTypeToLabel = {
   [ContentType.Raids]: {
     en: 'Raid',
     de: 'Raid',
+    ja: 'レイド',
+    cn: 'Raid',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:78]:
```suggestion
    cn: '大型任务',
```

File: util/coverage/coverage.js [hunk 2]
```diff
@@ -47,41 +47,59 @@ const exVersionToShortName = {
...
   },
   [ContentType.Trials]: {
     en: 'Trial',
     de: 'Prfng',
+    ja: '討伐戦',
+    cn: '蛮神',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:84]:
```suggestion
    cn: '讨伐歼灭战',
```

File: util/coverage/coverage.js [hunk 3]
```diff
@@ -47,41 +47,59 @@ const exVersionToShortName = {
...
   [ContentType.UltimateRaids]: {
     en: 'Ult',
     de: 'Ult',
+    ja: '絶',
+    cn: '绝',
   },
   [ContentType.Dungeons]: {
     en: 'Dgn',
     de: 'Dgn',
+    ja: 'ダンジョン',
```
Comment (MaikoTan) [inline util/coverage/coverage.js:95]: @r4ha Is there any abbreviation of Dungeon in Japanese? This should be shorten as possible.

File: util/coverage/coverage.js [hunk 4]
```diff
@@ -47,41 +47,59 @@ const exVersionToShortName = {
...
     en: 'Trial',
     de: 'Prfng',
+    ja: '討伐戦',
+    cn: '蛮神',
   },
   [ContentType.UltimateRaids]: {
     en: 'Ult',
     de: 'Ult',
+    ja: '絶',
+    cn: '绝',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:90]:
```suggestion
    cn: '绝境战',
```

File: util/coverage/coverage.js [hunk 5]
```diff
@@ -47,41 +47,59 @@ const exVersionToShortName = {
...
   },
   [ContentType.Dungeons]: {
     en: 'Dgn',
     de: 'Dgn',
+    ja: 'ダンジョン',
+    cn: '四人本',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:96]:
```suggestion
    cn: '迷宫挑战',
```

File: util/coverage/coverage.js [hunk 6]
```diff
@@ -98,22 +116,32 @@ const zoneGridHeaders = {
   expansion: {
     en: 'Ex',
     de: 'Ex',
+    ja: 'パッチ',
+    cn: '大版本',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:120]:
```suggestion
    cn: '资料片',
```

File: util/coverage/coverage.js [hunk 7]
```diff
@@ -124,16 +152,22 @@ const miscStrings = {
   title: {
     en: 'Cactbot Content Coverage',
     de: 'Cactbot Inhaltsabdeckung',
+    ja: 'Cactbot コンテンツ完成度',
+    cn: 'Cactbot 副本覆盖率',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:156]:
```suggestion
    cn: 'Cactbot 内容检视',
```

File: util/coverage/coverage.js [hunk 8]
```diff
@@ -124,16 +152,22 @@ const miscStrings = {
...
   },
   // Overall label for the expansion table.
   overall: {
     en: 'Overall',
     de: 'Insgesamt',
+    ja: '概要',
+    cn: '概要',
```
Comment (ShadyWhite) [inline util/coverage/coverage.js:163]:
```suggestion
    cn: '总览',
```

**Review by ShadyWhite — CHANGES_REQUESTED** [review]

File: util/coverage/coverage.js (hunk 3 above)
Comment (r4ha) [reply util/coverage/coverage.js:95]: ID (Instanced Dungeons) is good here I guess.

Comment (MaikoTan) [reply util/coverage/coverage.js:95]: Oh, I even forgot this!
Thank you!

File: util/coverage/coverage.js (hunk 7 above)
Comment (MaikoTan) [reply util/coverage/coverage.js:156]: "Content" is not 内容 here, it means 副本 or 玩法 in-game.

File: util/coverage/coverage.js (hunk 2 above)
Comment (MaikoTan) [reply util/coverage/coverage.js:84]: This is too long, maybe "讨伐战" is enough

File: util/coverage/coverage.js (hunk 7 above)
Comment (ShadyWhite) [reply util/coverage/coverage.js:156]: But put them together, then they are '内容'.It means 'cactbot中所包含的内容'.

**Pull request merged**